src/Evolution/Systems/CurvedScalarWave/BoundaryConditions/ConstraintPreservingSphericalRadiation.hpp

```diff
@@ -41,35 +41,78 @@ namespace CurvedScalarWave::BoundaryConditions {
  * are still likely to occur.
  *
  * The constraint-preserving part of the boundary conditions are set on the time
- * derivatives of the evolved fields according to \cite Holst2004wt . The
- * physical Bayliss-Turkel boundary conditions are additionally set onto the
- * time derivative of \f$\Pi\f$.
+ * derivatives of all evolved fields. The physical Bayliss-Turkel boundary
+ * conditions are additionally set onto the time derivative of \f$\Pi\f$.
  *
  * The constraints are defined as follows:
  *
  * \f{align*}{
  *  \mathcal{C}_i&=\partial_i\Psi - \Phi_i=0, \\
  *  \mathcal{C}_{ij}&=\partial_{[i}\Phi_{j]}=0
  * \f}
+ * Inspection of the constraint evolution system (Eqs. 29-30 in
+ * \cite Holst2004wt) shows that the constraints themselves are characteristic
+ * fields. We can derive constraint boundary conditions the same way
+ * \cite Kidder2004rw does for the Einstein equations:
  *
- * The boundary conditions are then given by:
+ * We express the constraints in terms of (evolution) characeristic fields and
+ * argue that the normal component of the constraint has to be zero when
+ * flowing into the boundary i.e. there are no constraints flowing into our
+ * numerical domain:
  *
  * \f{align*}{
- * \partial_{t} \Psi&\to\partial_{t}\Psi +
- *                    \lambda_\Psi n^i \mathcal{C}_i, \\
- * \partial_{t}\Pi&\to\partial_{t}\Pi-\left(\partial_t\Pi
- *                  - \partial_t\Pi^{\mathrm{BC}}\right)
- *                  +\gamma_2\lambda_\Psi n^i \mathcal{C}_i
- *                  =\partial_t\Pi^{\mathrm{BC}}
- *                  +\gamma_2\lambda_\Psi n^i \mathcal{C}_i, \\
- * \partial_{t}\Phi_i&\to\partial_{t}\Phi_i+
- *                     \lambda_0n^jP^k{}_i\mathcal{C}_{jk}
- *   = \partial_{t}\Phi_i+ \lambda_0n^j \mathcal{C}_{ji}.
+ * n^i \mathcal{C}_i &= n^i \partial_i Z^1 - \frac{1}{2}(U^{+} - U^-) + n^i
```

Comment:
Flat SW uses `w^\Psi`, `w^0_i` `w^\pm` so let's be consistent with that. This is also the notation that the Rezzolla book uses, and is quite standard in the engineering literature. It's basically only SpEC that uses this notation.